proto/sdv/databroker/v1/types.proto

```diff
@@ -143,16 +143,27 @@ message Datapoint {
   }
 }
 
+message Allowed {
+  oneof values {
+    StringArray string_values = 1;
+    Int32Array int32_values   = 3;
+    Int64Array int64_values   = 4;
+    Uint32Array uint32_values = 5;
+    Uint64Array uint64_values = 6;
+    FloatArray float_values   = 7;
+    DoubleArray double_values = 8;
+  }
+}
+
 message Metadata {
-  // Id to be used in "get" and "subscribe" requests. Ids stay valid during
-  // one power cycle, only.
   int32 id               = 1;
   EntryType entry_type   = 2;
   string name            = 4;
   DataType data_type     = 5;
   ChangeType change_type = 6;  // CONTINUOUS or STATIC or ON_CHANGE
   string description     = 7;
 
+  Allowed allowed = 10;
```

Comment:
Have we ever used the metadata on the lines below? Even if the lines below are commented I assume it would be better to not have the same number (10) on this line and the line below, or?

Reply:
They have never been used. I guess it's better to just remove them.

Reply:
Then just remove them I would say